src/Core/Sts/Tests/Integration/StsClientTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace AsyncAws\Core\Sts\Tests\Integration;
+
+use AsyncAws\Core\Sts\Input\AssumeRoleRequest;
+use AsyncAws\Core\Sts\Input\GetCallerIdentityRequest;
+use AsyncAws\Core\Sts\Input\PolicyDescriptorType;
+use AsyncAws\Core\Sts\Input\Tag;
+use PHPUnit\Framework\TestCase;
+
+class StsClientTest extends TestCase
+{
+    public function testAssumeRole(): void
+    {
+        self::markTestSkipped('Not implemented');
```

Comment:
Are we doing this because we want to manually activate them?

Reply:
Yes, I hesitate between `markTestSkipped` and `thowing an exception to force developper..

Reply:
I would probably remove tests if an exception was thrown..